experiments/HSA_20150714_MI/hsa_MI_20150714.py

```diff
@@ -0,0 +1,291 @@
+#! /usr/bin/env python
+"""
+Script for generation of input files for Aspirin and Naproxen binding to HSA by ITC.
+"""
+
+from itctools.itctools import ureg, Quantity
+from itctools.procedures import ITCProtocol, ITCExperimentSet, ITCExperiment, ITCHeuristicExperiment
+from itctools.materials import Solvent, Compound, SimpleSolution
+from itctools.labware import Labware, PipettingLocation
+import pprint
+
+
+## Is this volume correct? I can try adding units.
+
+# The sample cell volume in microliters
+cell_volume = 202.8
+
+# Define solvents.
+water = Solvent('water', density=0.9970479 * ureg.gram / ureg.milliliter)
+buffer = Solvent('buffer', density=1.014 * ureg.gram / ureg.milliliter) # TODO is our density the same as the HOST-GUEST buffer?
+
+# Define compounds.
+hsa = Compound('HumanSerumAlbumin', molecular_weight=65000 * (ureg.gram / ureg.mole), purity=.95)
+naproxen_sodium = Compound('Naproxen Sodium', molecular_weight=252.24 * (ureg.gram / ureg.mole), purity=1.)
+
+
+## Aspirin can be insoluble? Are we using its salt?
+
+# These are insoluble in water without added DMSO
+aspirin = Compound('AcetylsalicylicAcid', molecular_weight=180.15742 * (ureg.gram / ureg.mole), purity=.99)
+# naproxen = Compound('Naproxen', molecular_weight=230.3 * (ureg.gram / ureg.mole), purity=.98)
+
+#Ka (association constants) TODO Add this to the compound properties? (maybe a dict with protein as key)
+aspirin_ka = 547198.10 / ureg.molar  # http://omicsonline.org/2157-7544/2157-7544-2-107.pdf first site estimate
+naproxen_ka = 1.7 / (10 * ureg.micromolar)  # http://pubs.acs.org/doi/pdf/10.1021/jp062734p
+
+# Define troughs on the instrument
+water_trough = Labware(RackLabel='Water', RackType='Trough 100ml')
+buffer_trough = Labware(RackLabel='Buffer', RackType='Trough 100ml')
+
+# Define source labware.
+source_plate = Labware(RackLabel='SourcePlate', RackType='5x3 Vial Holder')
+
+# Define source solutions in the vial holder
+# TODO : Define solutions once prepared with the Quantos
+# hsa_solution = SimpleSolution(compound=hsa, compound_mass=13.8 * ureg.milligram, solvent=buffer, solvent_mass=0.5 * ureg.gram, location=PipettingLocation(
+#     source_plate.RackLabel,
+#     source_plate.RackType,
+#     1))
+
+## what is compound mass?
+## I updated compound mass and solvent mass values based on Bas's Dec 17, 2015 experiment. (hsa_short.py)
```

Comment:
We enter rough targets in the initial version, then update these values once we know the dispensed quantities.